test/acceptance/features/steps/openshift.py

```diff
@@ -334,3 +335,25 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_data(self, secret_name, secret_key, namespace, secret_value):
```

Comment:
Will this method be useful to any other test as well, if not, then why not to add this block of code in you step itself?

We would like to keep openshift related methods here. Can calling `get_resource_info_by_jsonpath` in your step itself works? WDYT?

Reply:
It will be used for #640